### DISCUSSION TITLE
Issue with importing EBSD data with .crc and .cpr extensions in MTEX

### DISCUSSION THREAD

**Discussion by Jingfan11** [mtex-toolbox/mtex#1281] [Ask Anything]
Hello,

I met some troubles on opening CRC and CPR files (I put them in the same folder) by MTEX. I always got "Could not detect file format..." Could anyone help me with it?

Thanks a lot!

Jingfan

**Comment by lfgmorales** [comment]
Dear Jingfan,

If you have access to your EBSD computer with Oxford Aztec or Channel, export your files as *.ctf, the CRC and CPR are Oxford files and the format may change from time to time

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 13 Nov 2021, at 06:27, Jingfan11 ***@***.***> wrote:

﻿

Hello,

I met some troubles on opening CRC and CPR files (I put them in the same folder) by MTEX. I always got "Could not detect file format..." Could anyone help me with it?

Thanks a lot!

Jingfan

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1281>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INYE73H76E36R3TBYG3ULXZLTANCNFSM5H6HKWLQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1281", "url": "https://github.com/mtex-toolbox/mtex/discussions/1281", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
